### DISCUSSION TITLE
'Official' Preview Package for .NET 8

### DISCUSSION THREAD

**Discussion by spencerdahl** [CommunityToolkit/Maui#1406] [General]
Will there be an 'Official' Preview Package for .NET 8?

**Comment by VladislavAntonyuk** [comment]
no, we don't plan to release a preview package.
but you can get the "official" nuget package previews from the PR: https://github.com/CommunityToolkit/Maui/pull/1367

**Reply by codeaphex** [reply]
Is this somehow blocked, or do I have to build the package manually?
It looks like builds should be pushed to azure but I cant find it on https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-MainLatest/nuget/v3/index.json  
The MVVM package seems to be available for .NET 8.

**Reply by VladislavAntonyuk** [reply]
You need to download zip archive with nuggets from the PR. We are preparing guide with steps

**Reply by TheCodeTraveler** [reply]
👆 @CliffAgius FYI

**Reply by codeaphex** [reply]
Ah thanks for the quick reply, finally found it, had to dig one step deeper. 
Azure pipeline - > Details on Azure - > Artifacts.

**Comment by SpenceDahl** [comment]
> You need to download zip archive with nuggets from the PR. We are preparing guide with steps

When and where will the guide be available?

**Reply by bijington** [reply]
It'll be going up on our docs pages at https://learn.microsoft.com/dotnet/communitytoolkit/maui/

That's the where. As to when I'm not entirely sure

**Comment by CliffAgius** [comment]
Guide added to the Wiki - https://github.com/CommunityToolkit/Maui/wiki/Preview-Packages

Closed and answered
